app/src/main/kotlin/com/mapbox/vision/teaser/utils/FragmentUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.vision.teaser.utils
+
+import androidx.fragment.app.Fragment
+import com.mapbox.vision.VisionManager
+import com.mapbox.vision.VisionReplayManager
+import com.mapbox.vision.manager.BaseVisionManager
+import com.mapbox.vision.teaser.MainActivity
+
+fun Fragment.requireVisionManager(): BaseVisionManager? {
```

Comment:
So it will return false if host is not MainActivity, right?

Reply:
It will return null

Reply:
And fragment fires backPress then. Maybe add some VisionLog.e to indicate an error? Or even throw an exception since it's unsupported at the moment

Reply:
good idea, I will add an exception to this method